### PR TITLE
Expose DD_SERVERLESS_LOGS_ENABLED env var, bump deps

### PR DESCRIPTION
**Pull request by astuyve**
### What does this PR do?

1. Bumps critical deps via `yarn audit --fix`
2. honors `enableDDLogs` in config and `DD_SERVERLESS_LOGS_ENABLED` in environment:
image
(I also verified that it's false when specified)

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [x] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

**Comment by agocs** [comment]
Looks great so far. I think all that's missing is documenting `enableDDLogs` in the README.md. That change should automatically be reflected in the [docs page](https://docs.datadoghq.com/serverless/libraries_integrations/plugin/).

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "test:watch": "jest --watch",
     "coverage": "jest --coverage",
     "lint": "tslint --project tsconfig.json",
-    "check-formatting": "prettier --check src/**",
-    "format": "prettier --write src/**"
+    "check-formatting": "prettier --check \"src/**\" \"README.md\"",
+    "format": "prettier --write \"src/**\" \"README.md\""
```
Comment (agocs) [inline package.json:17]: 👍

**Review by agocs — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -21,29 +21,30 @@ To quickly get started, follow the installation instructions for [Python][1] or
...
+| `enableDDTracing`           | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                                   |
+| `enableDDLogs`              | Enable Datadog log collection for the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                           |
```
Comment (nhinsch) [inline README.md:37]: Does this apply when using the Datadog Forwarder?

Comment (nhinsch) [reply README.md:37]: I think the answer to my own question is "no", that is, this setting only affects log collection if the customer is using the Lambda Extension, and not the older but still more popular Datadog Forwarder-based approach. I think we should document that here.

Comment (astuyve) [reply README.md:37]: I think you're correct, it doesn't apply to the Datadog Forwarder. Good call on documenting that here, will do

**Pull request merged**